lib/teiserver/lobby.ex

```diff
@@ -565,7 +565,8 @@ defmodule Teiserver.Lobby do
       Enum.any?([
         CacheUser.is_moderator?(user),
         Enum.member?(user.roles, "Caster"),
-        consul_reason == :override_approve
+        consul_reason == :override_approve,
+        consul_reason == :allow_friends
```

Comment:
You can combine these 2 conditions with `consul_reason in [:override_approve, :allow_friends]`

Reply:
alright, done